airbyte-integrations/connectors/source-jira/acceptance-test-config.yml

```diff
@@ -25,12 +25,10 @@ acceptance_tests:
         empty_streams:
           - name: "project_permission_schemes"
             bypass_reason: "Unable to populate. Jira doesn't support issue security for Free plan."
-        ignored_fields:
-          sprint_issues:
-            - name: updated
-              bypass_reason: "Unstable data"
-            - name: fields/updated
-              bypass_reason: "Unstable data"
+          - name: "issues"
```

Comment:
should we just ignore those fields? 

why can't we predict the values ​​for these custom fields?